app/models/topic.rb

```diff
@@ -86,7 +86,7 @@ class Topic < ActiveRecord::Base
   validates :name, presence: true, length: { maximum: 255 }
   # validates :user_id, presence: true
 
-  enum priority: { normal: 0, high: 1, very_high: 2 }
+  enum priority: { low: 0, normal: 1, high: 2, very_high: 3 }
```

Comment:
FWIW, you could just do `enum priority: [:low, :normal, :high, :very_high]` here. Rails will implicitly use 0 - 3 when storing the integer in the database. Definitely nothing wrong with declaring it explicitly though. :smile:

Reply:
I have experience of defining enum differently (increments of 10 instead of 1) in projects, used to define enum explicitly 😅 but let's see if it's acceptable this way.

Reply:
Right on 👍

Reply:
I think its better to delare specific integers in case you ever want to add some additional level you can.  See this blog post https://bendyworks.com/blog/rails-enum-sharp-knife

Reply:
@scott So is there anything I can improve from the current line? It's being explicitly defined for now and it's possible to put in other values in between the values